modules/ocf_mirrors/manifests/qt.pp

```diff
@@ -0,0 +1,19 @@
+class ocf_mirrors::qt {
+  file {
+    '/opt/mirrors/project/qt':
+      ensure  => directory,
+      source  => 'puppet:///modules/ocf_mirrors/project/qt/',
+      owner   => mirrors,
+      group   => mirrors,
+      mode    => '0755',
+      recurse => true;
+  }
+
+  cron {
+    'qt':
+      command => '/opt/mirrors/project/qt/sync-archive > /dev/null',
+      user    => 'mirrors',
+      hour    => '*/4',
+      minute  => '5';
```

Comment:
Also I'm surprised the ubuntu sync is only every 7 hours, their documentation says 6 is required, and they even support [push-triggered mirroring](https://wiki.ubuntu.com/Mirrors/PushMirroring), so maybe we could try that now that we opened up SSH access for Kali's push mirroring.